pages/get-started/index.tsx

```diff
@@ -43,17 +44,22 @@ const GetStarted: NextPage = () => {
           <button
             type="button"
             onClick={async () => {
-              await signIn("github", { callbackUrl: redirectTo });
+              try {
+                await signIn("github", { callbackUrl: redirectTo });
+                Fathom.trackGoal('sign-up-complete', 0);
+              } catch (e) {
+                console.error(e);
+              }
             }}
-            className="group relative w-full border border-transparent text-base bg-gradient-to-r from-orange-400 to-pink-600 rounded-md shadow-sm py-2 px-4 inline-flex justify-center font-medium text-white hover:from-orange-300 hover:to-pink-500 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-pink-300"
+            className="relative inline-flex justify-center w-full px-4 py-2 text-base font-medium text-white border border-transparent rounded-md shadow-sm group bg-gradient-to-r from-orange-400 to-pink-600 hover:from-orange-300 hover:to-pink-500 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-pink-300"
```

Comment:
This is a built in linter I have for tailwind this stuff is done automatically

Reply:
What is that linter? Looks handy!